apps/allocations/contracts/Allocations.sol

```diff
@@ -438,6 +441,28 @@ contract Allocations is AragonApp {
         }
     }
 
+    /**
+     * @param _accountId The account ID of the budget to be calculated
+     */
+    function getRemainingBudget(uint64 _accountId) public view returns (uint256) {
+        Account storage account = accounts[_accountId];
+        if (!account.hasBudget) {
+            return MAX_UINT256;
```

Comment:
If the account has no budget, it has the maximum budget possible?

Reply:
Yeah no budget means that the only limitation is an overflow. You might be thinking of a disabled budget, where  hasBudget is set to `true` and the budget value itself is set to 0